karate-core/src/main/java/com/intuit/karate/ExceptionUtils.java

```diff
@@ -0,0 +1,35 @@
+package com.intuit.karate;
+
+import java.io.PrintWriter;
+import java.io.StringWriter;
+
+public class ExceptionUtils {
+
+    private ExceptionUtils() {
+        // only static methods
+    }
+
+    /**
+     *
+     * From apache commons-lang3. If lib is ever used, this should be removed.
+     *
+     * Gets the stack trace from a Throwable as a String.
+     *
+     * <p>The result of this method vary by JDK version as this method
+     * uses {@link Throwable#printStackTrace(java.io.PrintWriter)}.
+     * On JDK1.3 and earlier, the cause exception will not be shown
+     * unless the specified throwable alters printStackTrace.</p>
+     *
+     * @param throwable  the {@link Throwable} to be examined, may be null
+     * @return the stack trace as generated by the exception's
+     * {@code printStackTrace(PrintWriter)} method, or an empty String if {@code null} input
+     */
+    public static String getStackTrace(final Throwable throwable) {
```

Comment:
there's already a `throwableToString()` in `StringUtils` so can you see if that is sufficient or adjust accordingly. I know `StringUtils` is not the best place for it - but I don't want too many utils

Reply:
Yep that produces the same outcome. Will delete this.